crud/replace.lua

```diff
@@ -30,15 +30,15 @@ function replace.init()
     })
 end
 
---- Insert or replace a tuple in the specifed space
+--- Insert or replace a tuple in the specified space
 --
--- @function call
+-- @function tuple
 --
 -- @param string space_name
 --  A space name
 --
--- @param table obj
---  Tuple object (according to space format)
+-- @param table tuple
+--  Tuple tuple
```

Comment:
What?

Reply:
what?

Reply:
Tuple tuple?

Reply:
Before it was `Tuple object` - was it OK?

Reply:
No, but below you use it:

```lua
-- @param table obj
--  Tuple object (according to space format)
```
Why not 

```lua
-- @param table obj
--  Object (according to space format)
```

```lua
-- @param table tuple
--  Tuple (according to space format)
```

??

Reply:
Stay just `Tuple`

Reply:
I don't understand what does it mean `Object  (according to space format)` if you mean field names how it related to `Tuple (according to space format)`. Tuple is just list of values.

Reply:
Ok, `Tuple` and `Object`.